### DISCUSSION TITLE
UCI SMS API

### DISCUSSION THREAD

**Discussion by ManojNathIC** [samagra-comms/community#8] [General Discussion]
In Shiksha, in the attendance module we have a situation where every day a dynamic list of absent students is generated to whom we need to send a message.

What have I tried till now:
I have read the documentation [here:](https://gist.github.com/ChakshuGautam/6a435756f09a586dd58ad0d6c5110b7d) showing the expected user segment expected response. 

Following this, I have created this sample response 

![image](https://user-images.githubusercontent.com/66711855/165057246-d7812af6-c6c3-4b64-9623-3c2d214166a0.png)

What I need:
So I need to send absent student notification to available phone number at the end of day. 
  @SaketSinha @surabhi-mahawar @ChakshuGautam @coolbung

**Comment by ChakshuGautam** [comment]
Hey @ManojNathIC @surabhi-mahawar, is the adapter ready for the SMS provider?

**Reply by SaketSinha** [reply]
@ChakshuGautam - aren't we using CDAC adapter only?

**Reply by ChakshuGautam** [reply]
If this is for POC only, sure we can use that only.

**Reply by ChakshuGautam** [reply]
Hey @ManojNathIC can you please share the public API with me along with the header? You can DM me on Discord. Thanks.

**Reply by SaketSinha** [reply]
Yes for now. Let's move in this direction.

**Reply by ManojNathIC** [reply]
> Hey @ManojNathIC can you please share the public API with me along with the header? You can DM me on Discord. Thanks.

@ChakshuGautam I've shared API link with you in DM

**Comment by SaketSinha** [comment]
Quoting @ChakshuGautam 

>  Hey these are the steps
> - Creating a user segment (@ManojNath  I will onboard your API to UCI) - just share the curl with me for it.
> - Creating a conversation logic and bot (Surabhi can help you with this)
> - Testing
